packages/jaeger-ui/src/components/TracePage/index.js

```diff
@@ -157,6 +162,9 @@ export class TracePageImpl extends React.PureComponent<TracePageProps, TracePage
       const { trace } = nextProps;
       this._scrollManager.setTrace(trace && trace.data);
     }
+    if (nextProps.mapCollapsed) {
```

Comment:
Can the `mapCollapsed` prop be used directly in the render function instead of using state? Would `!mapCollapsed && !slimView` work?

Reply:
mmm we can remove this now after the last change